application/src/utils/Percent.ts

```diff
@@ -0,0 +1,30 @@
+export default class Percent {
+  public static readonly Multiplier = 100;
+
+  public readonly value: number;
+  public readonly opposite: number;
+
+  constructor(value: number) {
+    this.value = value;
+    this.opposite = Percent.Multiplier - value;
+  }
+
+  static fromFraction(value: number): Percent {
+    return new Percent(value * this.Multiplier);
+  }
+
+  static fromList(list: number[]): Percent[] {
```

Comment:
can I enter any list of numbers to this?

Reply:
yes